packages/media-utils/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './components';
+
+export { uploadMedia } from './utils/upload-media';
```

Comment:
Compared to what was exported before:

https://github.com/WordPress/gutenberg/blob/23db6dcaef148526d7de0487fc5f2b09c45f2b90/packages/media-utils/src/utils/index.js#L1

Reply:
Probably it's better to export them as private APIs for now(lock/unlock). I don't think there is a reason to make them public right away, before the whole feature lands.

Reply:
Soo...shall I lock them? What about the exported types? Can't really make those private.

Reply:
We could, but on the other hand it feels like these utils can be used in general. I have no strong opinion on this..

Reply:
I don't expect them to have breaking changes